src/nix/repl.cc

```diff
@@ -611,11 +611,11 @@ void NixRepl::loadFile(const Path & path)
 
 void NixRepl::loadFlake(const std::string & flakeRefS)
 {
-    auto flakeRef = parseFlakeRef(flakeRefS, absPath("."), true);
+    auto [flakeRef, fragment] = parseFlakeRefWithFragment(flakeRefS, absPath("."), true);
     if (evalSettings.pureEval && !flakeRef.input.isImmutable())
         throw Error("cannot use ':load-flake' on mutable flake reference '%s' (use --impure to override)", flakeRefS);
 
-    Value v;
+    auto v = state->allocValue();
```

Comment:
What’s the reason for the change? Having `v` as a plain stack-allocated `Value` is nice since it has a lexical lifetime.

Reply:
Oh I think I copied that from somewhere else while try to debug a stack overflow. I'll change it back :+1: